components/eventsrv/src/subscriber.rs

```diff
@@ -41,10 +49,11 @@ fn main() {
                 let received_payload = String::from_utf8(event.get_payload().to_vec()).unwrap();
                 let member_id = event.get_member_id();
                 let timestamp = event.get_timestamp();
-
-                println!("Timestamp {}", timestamp);
-                println!("Member ID {}\n", member_id);
-                debug!("{}\n", received_payload);
+                let service = event.get_service();
+                println!("SUBSCRIBER: Timestamp {}", timestamp);
+                println!("SUBSCRIBER: Member ID {}", member_id);
+                println!("SUBSCRIBER: Service {}", service);
+                println!("SUBSCRIBER: Payload {}\n", received_payload);
```

Comment:
@raskchanky I was thinking about removing them; what's the project's general approach to logging?